src/utils/data_reader.py

```
@@ -0,0 +1,136 @@
+            file_path = self.manager.get_processed_path('terrain') / f"{country.lower()}_elevation_grid.csv"
+        
+        if not file_path.exists():
+            raise FileNotFoundError(f"No elevation data found for {country}")
+        
+        return pd.read_csv(file_path)
+    
+    def read_fires(self, country: str) -> pd.DataFrame:
+        files = self.manager.list_files('firms', f"firms_{country.lower()}_*")
+        
+        if not files:
+            raise FileNotFoundError(f"No fire data found for {country}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)